### PR TITLE
Feat: Rank command

### PR DESCRIPTION
**Pull request by juancastellano115**
Related to https://github.com/rokkoo/valorat-discord-bot/issues/1

- We need to get the data from this api https://docs.henrikdev.xyz/valorant.html
-  Createa command listener to handle the player id passed on the parameter params

### CODE REVIEW

File: src/commands/index.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { Message } from 'discord.js';
-
+import axios from 'axios';
```
Comment (rokkoo) [inline src/commands/index.ts]: Can we move this to another file so we can be more modular? Thanks!

File: src/commands/index.ts [hunk 2]
```diff
@@ -14,7 +14,41 @@ const helloCommand: Command = {
...
+const RankCommand: Command = {
+  command: 'rank',
+  async action(message, args) {
+  
+    if (args.length === 0) {
+     return message.reply(`Provide a username and tag! (Fulanin#EUW)`);
+    }
+    // Handling names with spaces
+    const rawArgs = args.join(' ');
+    const splittedArgs = rawArgs.split('#');
```
Comment (rokkoo) [inline src/commands/index.ts]: Can you create a lib folder and export there a function to by passing the 'username#tag' will return in an object the username and tag?

File: src/commands/index.ts [hunk 3]
```diff
@@ -14,7 +14,41 @@ const helloCommand: Command = {
...
+    // Encode in URL format
+    const name = encodeURI(splittedArgs[0]);
+    const tag = splittedArgs[1];
+
+    if (!name || !tag) {
+      return message.reply(`Provide a username and tag! (Fulanin#EUW)`);
+     }
+
+    const response = await axios.get(`https://api.henrikdev.xyz/valorant/v1/mmr/eu/${name}/${tag}`);
```
Comment (rokkoo) [inline src/commands/index.ts]: Just in case wrap in a try/catch so maybe the api is down or something?

Comment (rokkoo) [reply src/commands/index.ts]: Also I think is a good oportunity to create a const with the base api uri and use it, what do you think?
For example 
const API_VERSION = 'v1'
`const url='https://api.henrikdev.xyz/valorant/${API_VERSION}/'`

File: src/commands/index.ts (hunk 1 above)
Comment (juancastellano115) [reply src/commands/index.ts]: Done!

**Comment by juancastellano115** [comment]
@rokkoo I just added the changes you mentioned, take a look when you can.

Looks much more clean! ty man

**Comment by rokkoo** [comment]
Thanks @juancastellano115 looks cool, approved! thanks you!

**Pull request merged**